en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md

```diff
@@ -39,11 +54,35 @@ Let's assume you have two environments called "choreo-connect-1" and "choreo-con
         reconnectInterval = 5000
         reconnectRetryCount = 60
     ```
-        
-2. Start Choreo Connect deployment from docker-compose. 
 
-3. Add the Gateway environments "choreo-connect-1" and "choreo-connect-2" from the Gateways in the APIM admin portal.
+2. Configure gateway environment label in Enforcer and Router. Update the following environment labels in each component. Update the following environment variables defined in the docker-compose file.
+   
+   -  Enforcer
+      ``` tab="Format"
+      ENFORCER_LABEL=<GATEWAY_LABEL>
+      ```
+      ``` tab="Enforcer 1"
+      ENFORCER_LABEL=choreo-connect-1
+      ```
+      ``` tab="Enforcer 2"
+      ENFORCER_LABEL=choreo-connect-2
+      ```
+
+   -  Router
+      ``` tab="Format"
+      ROUTER_LABEL=<GATEWAY_LABEL>
+      ```
+      ``` tab="Enforcer 1"
+      ROUTER_LABEL=choreo-connect-1
+      ```
+      ``` tab="Enforcer 2"
+      ROUTER_LABEL=choreo-connect-2
+      ```
+
+3. Start Choreo Connect deployment from docker-compose.
+
+4. Add the Gateway environments "choreo-connect-1" and "choreo-connect-2" from the Gateways in the APIM admin portal.
```

Comment:
Are these environments or labels?

Reply:
environment labels, but in the admin portal below it is gateway environments